### PR TITLE
Search for a content with accented characters

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+               contentAdministration = new ContentAdministration(this);
+              navigationToolbar = new NavigationToolbar(this);
+               siteExplorerHome = new SiteExplorerHome(this);
+               createNewDoc= new CreateNewDocument(this);
+               manageLogInOut=new ManageLogInOut(this);
+                 if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+                     $(ELEMENT_SKIP_BUTTON).click();
+                   }
+        manageLogInOut.signInCas(username, password);
+}
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java]: Indentation is not good.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+     /*
+      bug ECMS-7688
+     */
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java]: Wrong format for comments on method, and we should add a description to directly see what this test is about (without going to jira).

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/ecms/ECMSLocator.java [hunk 3]
```diff
@@ -1147,4 +1147,11 @@
   public static final SelenideElement ELEMENT_LIST_FOLDER_IN_DEFAULT_VIEW=$(byId("UIDocumentWorkspace"));
 
   public static final SelenideElement ELEMENT_BUTTON_CONFIRM_DELETE_FILE=$(byClassName("UIDeleteFileConfirmMessage")).find(byClassName("OK"));
+  public static final SelenideElement ELEMENT_SEARCH_BTN=$(byId("simpleSearch"));
+  public static final SelenideElement ELEMENT_ICON_SEARCH=$(byClassName("uiIconSearch"));
+  public static final SelenideElement ELEMENT_TITLE_WEB_CONTENT=$(byClassName("title"));
+
+
+
+
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/ecms/ECMSLocator.java]: Use same formatting than others variables and remove useless blank lines please.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java (hunk 1 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java]: Done

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/ecms/ECMSLocator.java (hunk 3 above)
Comment (aboughdiri) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/ecms/ECMSLocator.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java (hunk 2 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+   createNewDoc.saveAndClose();
+   navigationToolbar.goToSiteExplorer();
+   siteExplorerHome.goToPath("intranet", "Site Management");
+   ELEMENT_SEARCH_BTN.setValue("sécurité").pressEnter();
+   $(byId("SimpleSearchResult")).find(byXpath("//*[@id=\"SimpleSearchResult\"]/table/tbody/tr[1]/td[2]")).find(byText("sécurité")).shouldBe(Condition.visible);
+   info("Delete web content");
+   navigationToolbar.goToSiteExplorer();
+   siteExplorerHome.goToPath("intranet", "Site Management");
+   siteExplorerHome.deleteData("sécurité");
+      }
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java]: Indentation is not good. Please use eXo formatting configuration file : http://developer.exoplatform.org/#id-ide-settings-to-work-on-exo-platform-projects

Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSSearchWebContentWithAccentedCharactersTestIT.java]: done

**Pull request merged**